app/lua/http.go

```diff
@@ -423,8 +424,21 @@ func CreateRequestClient(L *glua.LState) int {
 		return 0
 	}
 
+	// Header holder
+	headers := L.NewTable()
+
+	// Loop response header
+	for k, v := range resp.Header {
+
+		// Set header
+		headers.RawSetString(k, glua.LString(strings.Join(v, ", ")))
```

Comment:
Instead of `strings.Join` what about

```go
headers := L.NewTable()
	
for k, v := range resp.Header {

	if len(v) > 1 {
			
		h := L.NewTable()
			
		for _, header := range v {
				
			h.Append(glua.LString(header))
		}
		
		headers.RawSetString(k, h)
		
		continue
	}
	
	headers.RawSetString(k, glua.LString(v[0]))
}
```